_posts/2022-01-14-hhvm-4.144.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: "HHVM 4.144"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.144 is released! This release marks the end of support of 4.136;
+4.137, 4.138, 4.139, 4.140, 4.143 remain supported, as do the 4.102 and 4.128 LTS releases.
+4.141 and 4.142 were cancelled due to the holiday season.
+
+
+# Highlights
+
+- The `--config type_printer_fuel=N` option is introduced to `hh_client` to truncated error message when printing out a type, in case of hanging the type checker if the type is complicated.
+- `classname<E>` where `E` is an enum now supports dynamic
+
+# Breaking Changes
+
+- Local coeffect is enforced at runtime (previously coeffect error was only reported in the type checker, not runtime).
```

Comment:
Is this a coeffect called `local` , or 'local coeffects'? Maybe an example would be useful

Reply:
Updated. It should be local coeffects.

Reply:
> Maybe an example would be useful

The behavior should be unchanged unless the user runs a program with type checking errors.